src/main/scala/chisel3/util/Bitwise.scala

```diff
@@ -24,13 +26,27 @@ object FillInterleaved {
     *
     * Output data-equivalent to in(size(in)-1) (n times) ## ... ## in(1) (n times) ## in(0) (n times)
     */
-  def apply(n: Int, in: UInt): UInt = apply(n, in.asBools)
+  final def apply(n: Int, in: UInt): UInt = macro SourceInfoTransform.nInArg
```

Comment:
i realized i was inconsistent in marking these as `final def`. Is that correct/required if these are `object`s?

Reply:
I don't know what difference it makes on `object`s, trying to figure that out

Reply:
I asked on Scala gitter. It doesn't make any difference to use `final` and thus it's discouraged since doing it suggests that it _does_ mean something to anyone reading the code.

Reply:
cleaned this up.